packages/mui-varnish/src/colors.ts

```diff
@@ -0,0 +1,163 @@
+  G8: new Color('G8', '#0A8F6B', true),
+  G9: new Color('G9', '#00755A', true),
+  G10: new Color('G10', '#005340', true),
+  T1: new Color('T1', '#E6FDFE'),
+  T2: new Color('T2', '#C6F3F6'),
+  T3: new Color('T3', '#9AE7EC'),
+  T4: new Color('T4', '#6EDCE3'),
+  T5: new Color('T5', '#42D0D9'),
+  T6: new Color('T6', '#16C4CF', true),
+  T7: new Color('T7', '#0FA9B6', true),
+  T8: new Color('T8', '#078E9E', true),
+  T9: new Color('T9', '#007385', true),
+  T10: new Color('T10', '#004752', true),
+  A1: new Color('A1', '#F2FCFF'),
+  A2: new Color('A2', '#E0F9FF'),
+  A3: new Color('A3', '#B5F0FF'),
+  A4: new Color('A4', '#85E9FF'),
+  A5: new Color('A5', '#4DE1FF'),
+  A6: new Color('A6', '#00D5FF', true),
+  A7: new Color('A7', '#00C1E8', true),
+  A8: new Color('A8', '#01A2CA', true),
+  A9: new Color('A9', '#0278A7', true),
+  A10: new Color('A10', '#054976', true),
+  B1: new Color('B1', '#F0F7FF'),
+  B2: new Color('B2', '#D5EAFE'),
+  B3: new Color('B3', '#80BDFF'),
+  B4: new Color('B4', '#2F85F7'),
+  B5: new Color('B5', '#2376E5'),
+  B6: new Color('B6', '#265ED4', true),
+  B7: new Color('B7', '#1A4CAE', true),
+  B8: new Color('B8', '#1B4596', true),
+  B9: new Color('B9', '#1D3D7E', true),
+  B10: new Color('B10', '#223367', true),
+  P1: new Color('P1', '#F8F7FD'),
+  P2: new Color('P2', '#E6E3F7'),
+  P3: new Color('P3', '#CFC9F1'),
+  P4: new Color('P4', '#B7AFEB'),
+  P5: new Color('P5', '#A094E4'),
+  P6: new Color('P6', '#887ADE', true),
+  P7: new Color('P7', '#7265C1', true),
+  P8: new Color('P8', '#5C50A4', true),
+  P9: new Color('P9', '#463B87', true),
+  P10: new Color('P10', '#271F55', true),
+  M1: new Color('M1', '#FDF7FC'),
+  M2: new Color('M2', '#F6DFF3'),
+  M3: new Color('M3', '#EFC0E8'),
+  M4: new Color('M4', '#E7A2DE'),
+  M5: new Color('M5', '#E083D3'),
+  M6: new Color('M6', '#D864C9', true),
+  M7: new Color('M7', '#BE54B0', true),
+  M8: new Color('M8', '#A44397', true),
+  M9: new Color('M9', '#8A337E', true),
+  M10: new Color('M10', '#65295D', true),
+  N1: new Color('N1', '#FFFFFF'),
+  N2: new Color('N2', '#F8F9FA'),
+  N3: new Color('N3', '#F0F4F7'),
+  N4: new Color('N4', '#E8ECF2'),
+  N5: new Color('N5', '#D5DAE3'),
+  N6: new Color('N6', '#AEB7C4', true),
+  N7: new Color('N7', '#8C96A3', true),
+  N8: new Color('N8', '#616C7A', true),
+  N9: new Color('N9', '#47515C', true),
+  N10: new Color('N10', '#303945', true),
+  black: new Color('Black', '#000', true),
+  white: new Color('White', '#FFF'),
+  transparent: new Color('Transparent', 'transparent'),
+};
+
+export const lightCategoricalColor = {
+  Red: { ...color.R9, displayName: 'Red' },
+  Orange: { ...color.O9, displayName: 'Orange' },
+  Aqua: { ...color.A10, displayName: 'Aqua' },
+  Teal: { ...color.T8, displayName: 'Teal' },
+  Blue: { ...color.B6, displayName: 'Blue' },
+  Magenta: { ...color.M10, displayName: 'Magenta' },
+  Purple: { ...color.P7, displayName: 'Purple' },
+  Green: { ...color.G9, displayName: 'Green' },
+};
```

Comment:
I'm not sure what a categorical color is. I might just call this `lightColors` and `darkColors`, or something :).

Reply:
categorical/qualitative (blue, red, green, etc) is a common term used in charting colors to distinguish from liner/sequential (light blue to dark blue) or divergent (dark blue to dark red).